week1/task_Judging the shape of a triangle.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h> 
+#include <math.h>
+#include <conio.h>
+
+#define EPS 1e-1
+
+int main()
+{
+	while (1) { 
+		float a, b, c;
+		printf("请输入三角形的三边长：");
+		scanf("%f%f%f", &a, &b, &c);
+		if (a + b > c && a + c > b && b + c > a) {
+			if(fabs(a * a + b * b - c * c) <= EPS
+			|| fabs(a * a + c * c - b * b) <= EPS
+			|| fabs(b * b + c * c - a * a) <= EPS) {
+				printf("三角形为直角三角形\n");
+			}
+			else if (fabs(a - b) <= EPS

```

Comment:
I checked the Linux kernel repo. It seems that keep `}` and `else if` at the same line is a more usual way.